Задание_8.1/Задание_8.1/Program.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Задание_8._1
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.Write("Введите переменную: ");
+            int a, b, n;
+            a = Convert.ToInt32(Console.ReadLine());
```

Comment:
Лучше используйте метод `Parse()`
```suggestion
            a = int.Parse(Console.ReadLine());
```